src/app/pages/menu/users/user-details/user-details.page.ts

```diff
@@ -42,6 +42,10 @@ export class UserDetailsPage implements OnInit {
 
   }
 
+  addCard() {
```

Comment:
There is no need for an empty method, especially one designed to add a card in the user details page

Reply:
Probably started down a path that I realized didn't need to be traveled.

Reply:
Then please remove it!